cardano-cli/src/Cardano/CLI/Shelley/Run/Address.hs

```diff
@@ -91,8 +91,8 @@ writePaymentKeyFiles
   -> ExceptT ShelleyAddressCmdError IO ()
 writePaymentKeyFiles (VerificationKeyFile vkeyPath) (SigningKeyFile skeyPath) vkey skey = do
   firstExceptT ShelleyAddressCmdWriteFileError $ do
-    newExceptT $ writeFileTextEnvelope skeyPath (Just skeyDesc) skey
```

Comment:
We don't need to break the previous interface. Separating out IO is useful but lets leave the functions as they are now.